entity-framework/core/modeling/dynamic-model.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Dynamic Models - EF Core
+author: ansvyryd
+
+uid: core/modeling/dynamic-model
+---
+# Dynamic Models
+
+The model built in `OnModelCreating` could use a property on the context to change how the model is built. For example it could be used exclude a certain property:
+
+<!-- [!code-csharp[Main](samples/core/DynamicModel/DynamicContext.cs?range=6-31)] -->
```

Comment:
You should be able to uncomment this code reference and get rid of the snippet below. Also, consider relying on regions (e.g. DynamicContext.cs?#Class) rather than on line numbers because the latter are easier to break.